src/style.css

```diff
@@ -1,3 +1,61 @@
 body {
   margin: 0;
+  background-color: #f4f4f4;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic family as a fallback, such as 'sans-serif' or 'serif'. In this case, you have 'Roboto' and 'Arial', which are both sans-serif fonts, but it's always a good idea to include 'sans-serif' at the end as well to cover all bases.